database/migrations/2023_02_09_152218_update_entity_events_add_recurring_until_month_recurring_until_day.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('entity_events', function (Blueprint $table) {
+            $table->unsignedMediumInteger('recurring_until_month')->nullable();
```

Comment:
Since these fields are used in a `where()` query, we need to index them to avoid horrible performance on prod. Each should have its own index.